app/web/features/messages/groupchats/CreateGroupChat.tsx

```diff
@@ -136,31 +133,54 @@ export default function CreateGroupChat({ className }: { className?: string }) {
                 className={classes.field}
               />
             )}
-            <Controller
-              control={control}
-              name="users"
-              render={({ onChange, value }) => {
-                return (
-                  <Autocomplete
-                    id="users-autocomplete"
-                    onChange={(_, newValue) => {
-                      onChange(newValue);
-                      setIsGroup((newValue?.length ?? 0) > 1);
-                    }}
-                    multiple={true}
-                    loading={friends.isLoading}
-                    options={friends.data ?? []}
-                    noOptionsText={COULDNT_FIND_ANY_FRIENDS}
-                    getOptionLabel={(friend) => {
-                      return friend?.name ?? ERROR_USER_LOAD;
-                    }}
-                    label={FRIENDS}
-                    className={classes.field}
-                    value={value ?? []}
-                  />
-                );
-              }}
-            />
+            {createMessageToUserQuery.error && (
+              <Alert severity="error">{createMessageToUserQuery.error}</Alert>
+            )}
+            {
+              // need to mount the autocomplete with the correct default value
+              // of the "to" user, display a dummy loader until then
```

Comment:
So this is effectively bailing out on the server side and let the hydrated app deal with it later in client side?

Reply:
No (well, yes, that data isn't in the server render), it's just before the whole `createMessageTo` user was being passed in the router state, now we have to load from the id in the url. But you can't mount the `Control` and change the default value once the user data is loaded, so I'm actually starting with a dummy loading autocomplete (L174) and swapping for the proper one with a `Control` and `defaultValue` once the data is loaded.

Reply:
Could we not do the data loading of the user on server side too, since it'll have access to the id from the URL parameter? Or maybe too much change to this otherwise even if it works?